src/Numerical/Spectral/MeanValue.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function MeanValue.
+
+#pragma once
+
+#include "Domain/Side.hpp"
+#include "Numerical/Spectral/DefiniteIntegral.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+/*! \ingroup Functors
+ * \brief Compute the mean value of a grid-function over a manifold.
+ *
+ * \remarks The mean value is computed on the reference element(s).
+ *
+ * \returns the mean value of f on the manifold
+ */
+template <size_t Dim>
+double mean_value(const Scalar<DataVector>& f, const Index<Dim>& extents) {
+  return Basis::lgl::definite_integral(f, extents) / two_to_the(Dim);
+}
+
+/*! \ingroup Functors
+ *
+ * Compute the mean value of a grid-function on a boundary of a manifold.
+ * \f$mean value = \int f dV / \int dV\f$
```

Comment:
Shouldn't this equation be in the documentation of mean_value as well?  It's easier to understand there.

Reply:
👍